src/components/productoverview/ProductSorting.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <select data-test="sorting-select" name="sort" id="sort" v-model="sortBy">
```

Comment:
we usually put `data-tets` `select` `name` etc... underneath each other

Reply:
Source: https://vuejs.org/v2/style-guide/#Multi-attribute-elements-strongly-recommended